### PR TITLE
ISS-114 # Load test - Pure JUnit with zerocode report

### PR DESCRIPTION
**Pull request by nirmalchandra**

### CODE REVIEW

File: src/main/java/org/jsmart/zerocode/core/runner/ZeroCodeUnitRunner.java [hunk 1]
```diff
@@ -96,56 +105,21 @@ public void run(RunNotifier notifier) {
...
 
-            if (passed) {
-                LOGGER.info(String.format("\n**FINISHED executing all Steps for [%s] **.\nSteps were:%s",
-                        child.getScenarioName(),
-                        child.getSteps().stream().map(step -> step.getName()).collect(Collectors.toList())));
-            }
+        } else if (annotation != null) {
 
-            notifier.fireTestFinished(description);
+            runLeafJsonTest(notifier, description, annotation);
```
Comment (authorjapps) [inline src/main/java/org/jsmart/zerocode/core/runner/ZeroCodeUnitRunner.java]: 👍

File: pom.xml [hunk 2]
```diff
@@ -51,10 +51,13 @@
     </distributionManagement>
 
     <properties>
-        <!-- Build will not fail if error occurs during javadoc generation -->
-        <!--<maven.javadoc.failOnError>false</maven.javadoc.failOnError>-->
         <!-- Build will skip the javadoc generation -->
-        <maven.javadoc.skip>true</maven.javadoc.skip>
+        <!--<maven.javadoc.skip>true</maven.javadoc.skip>-->
```
Comment (dinesh76in) [inline pom.xml]: can delete these if not needed

Comment (nirmalchandra) [reply pom.xml]: will check if validation rules passes this time in Maven Central release task. After that, these can be cleared.

File: src/main/java/org/jsmart/zerocode/core/runner/ZeroCodeUnitRunner.java (hunk 1 above)
Comment (authorjapps) [reply src/main/java/org/jsmart/zerocode/core/runner/ZeroCodeUnitRunner.java]: No tests were found to make sure pure `JUnit tests` was executed. Please add a happy scenario

Comment (nirmalchandra) [reply src/main/java/org/jsmart/zerocode/core/runner/ZeroCodeUnitRunner.java]: yes, added one.
PR merged fyi. Please take care of the release to maven central.

**Pull request merged**